### PR TITLE
FIX: Euro symbol for  iOS Today extension

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6vya732x2r8wr2pdg33k9chqt0

[download apk](http://filestorage.bluewallet.io:1488/1592188396.apk)

File: models/fiatUnit.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const FiatUnit = Object.freeze({
   CLP: { endPointKey: 'CLP', symbol: '$', locale: 'es-CL' },
   CZK: { endPointKey: 'CZK', symbol: 'Kč', locale: 'cs-CZ' },
   CNY: { endPointKey: 'CNY', symbol: '¥', locale: 'zh-CN' },
-  EUR: { endPointKey: 'EUR', symbol: '€', locale: 'en-EN' },
+  EUR: { endPointKey: 'EUR', symbol: '€', locale: 'en-IE' },
```
Comment (Overtorment) [inline models/fiatUnit.js:10]: ireland?

Comment (Overtorment) [reply models/fiatUnit.js:10]: how s that supposed to work

Comment (marcosrdz) [reply models/fiatUnit.js:10]: The .locale is used for NumberFormatter to obtain the currency symbol. There en-en didn’t return the EUR symbol for numberformatter.

**Pull request merged**